### DISCUSSION TITLE
Bug converting open issues to discussions?

### DISCUSSION THREAD

**Discussion by robgjansen** [community/community#2944] [Discussions] [answered]
I just tried converting over 200 open issues in bulk on [shadow/shadow](https://github.com/shadow/shadow) into discussions following this process:
- navigate to [the labels page](https://github.com/shadow/shadow/labels)
- click the "convert issues" button
![Screen Shot 2020-12-10 at 1 52 49 PM](https://user-images.githubusercontent.com/788144/101816591-34a8bb00-3aef-11eb-9b5d-f7ef908901eb.png)

The issues have all been removed from the issue tracker and the label associated with them was empty (so I deleted the label). But now my discussions page has only ~30ish discussions even though I had converted over 200 issues. So it appears like most of the issues are missing.

Is this a bug, or is there simply a task in the queue and they will convert eventually?

I imagine this is important as more people start using the Discussions feature and converting old issues.

**Comment by robgjansen** [comment, marked as answer]
Hmm, the plot thickens. I've just discovered that the missing issues had their labels cleared and then were actually closed instead of being transferred to discussions. Indeed, the missing issues are still in [the issue tracker](https://github.com/shadow/shadow/issues?q=is%3Aissue+author%3Ashadow-org-bot).

It would be nice to have an easy way to convert CLOSED issues into discussions, but that doesn't seem to be possible.

**Reply by becca** [reply]
When you `close` an issue are you hoping most of the conversations halt or go elsewhere/somewhere new? If so, does that expectation change if a closed issue is moved into a Discussion?

**Reply by robgjansen** [reply]
I think the conversations would halt initially. But when people search that question again later and come to a closed GH issue, I think they would be unlikely to reopen it and/or continue the conversion (a closed issue feels "final"). On the other hand, a discussion feels like it could be more ongoing, and one with an already chosen answer is more likely to be revived in the Discussions system than a closed issue in the Issue system. Therefore, I was planning to move all old questions that were previously closed into the Discussion system. Being able to convert closed issues into discussions would help with this task.

**Reply by becca** [reply]
Thank you so much for this feedback. We are now tracking this feature request and will do our best to keep you posted on when this feature lands. Also, be sure to keep you eye on our [public roadmap](github.com/github/roadmap) for more updates!
